libs/core/src/lib/select/select-key-manager.service.ts

```diff
@@ -1,8 +1,8 @@
 import { Injectable } from '@angular/core';
 import { takeUntil } from 'rxjs/operators';
 import { ActiveDescendantKeyManager } from '@angular/cdk/a11y';
-import { KeyUtil } from '@fundamental-ngx/core/utils';
-import { UP_ARROW, DOWN_ARROW, ENTER, SPACE, HOME, END, hasModifierKey } from '@angular/cdk/keycodes';
+import { KeyUtil } from '../utils/functions';
```

Comment:
Why `@fundamental-ngx/core/utils` is replaced by `../utils/functions`?

Reply:
replaced.